scripts/linux/psv/travis_test_psv.sh

```diff
@@ -1,23 +1,25 @@
-#!/bin/bash -e
+#!/bin/bash
 CPP_TEST_SOURCE_AUTHENTICATION=build/olp-cpp-sdk-authentication/test
 CPP_TEST_SOURCE_CORE=build/olp-cpp-sdk-core/tests
 CPP_TEST_SOURCE_DARASERVICE_READ=build/olp-cpp-sdk-dataservice-read/test
 CPP_TEST_SOURCE_DARASERVICE_WRITE=build/olp-cpp-sdk-dataservice-write/test
+RC=0
```

Comment:
Is there any reason why we want to continue executing although we have a failure?
I don't see any reason. It is probably useful for yourself but not for the CI and I dont want to wait 30 mins to get a feedback that my first test failed.